.git-blame-ignore-revs

```diff
@@ -1,4 +1,6 @@
 # Exclude the following commits from the git blame -- see https://stackoverflow.com/a/69118451
 # You may need to run git config blame.ignoreRevsFile .git-blame-ignore-revs
 ## 2022-11-09 - Ran prettier code-formatter on entire repo after adding lint-staged to enforce code formatting
-53b439d81b4126848ce42514c6b5cc62fe5f5fa8
\ No newline at end of file
+53b439d81b4126848ce42514c6b5cc62fe5f5fa8
+## 2024-03-21 - Reverted prettier-plugin-java to v2.5.0 to avoid regression https://github.com/jhipster/prettier-java/issues/650
```

Comment:
A change in v2.6.0 changed a lot of formatting (for the worse IMO), so let's pin the version to v2.5.0 until it gets sorted out